experimental/framework/frs-client/package.json

```diff
@@ -26,6 +26,11 @@
     "eslint:fix": "eslint --format stylish src --fix",
     "lint": "npm run eslint",
     "lint:fix": "npm run eslint:fix",
+    "start:tinylicious:test": "tinylicious > tinylicious.log 2>&1",
+    "test:frs": "cross-env process.env.FLUID_CLIENT='\"frs\"' && npm run test:realsvc:run",
+    "test:realsvc:run": "mocha --unhandled-rejections=strict --recursive dist/test/**/*.spec.js --exit",
```

Comment:
could you add a `"test:realsvc"` script that runs the local (tinylicious) test? to match other tests that can vary their service

Reply:
I changed the `"test:realsvc"` to run the test against Tinylicious and `"test:realsvc:frs"` to run the test against FRS